cypress-E2E-tests/cypress/integration/device_properties_widget.cy.ts

```diff
@@ -366,8 +366,216 @@ describe("Device properties widget", function () {
+            fragment: fragment,
+            series: series,
+            unit: unit,
+            value: value
+        };
+        cy.createMeasurement(requestBody);
+      }
+    
+      cy.apiCreateSimpleAsset([assetObject]);
+      for (let i = 1; i < 3; i++) {
+        cy.createDevice(`Device${i}`);
+        cy.apiAssignDevice([`Device${i}`],asset);
+        createMeasurement(`Device${i}`, 's7aFlow', 'F', 'KL', 45);
+        createMeasurement(`Device${i}`, 's7aTemp', 'T', 'K', 450);
+      }
+    });
+
+    beforeEach(function () {
+      cy.login();
+      cy.visitAndWaitUntilPageLoad(url);
+      cy.get(asset_properties_widget_elements.addWidgetButton).click();
+      cy.get(asset_properties_widget_elements.cardElement).eq(0).click();
+      cy.clickOnAsset(asset);
+      cy.get(`p[title='${device}']`).should('contains.text', device);
+      cy.chooseAssetOrDevice(device);
+    });
+
+    // Function to add a data point configuration
+    const addDataPointConfiguration = () => {
+      cy.get(asset_properties_widget_elements.addPropertyButton).click();
+      cy.selectProperty(last_measurement_str);
+      cy.get(asset_properties_widget_elements.selectButton).click();
+      cy.get('.card-title').should('have.text', 'Data points');
+      cy.intercept({ method: 'GET', url: '/inventory/managedObjects?**)' }).as(
+        'assetselectionresponse'
+      );
+      cy.get(addDataPointConfigButton).click();
+      cy.wait('@assetselectionresponse')
+        .its('response.statusCode')
+        .should('eq', 200);
+      cy.clickOnAsset(asset,1);
+      cy.get('c8y-datapoint-selector-modal #modal-title').should('contain.text','Data point selector');
+      cy.chooseAssetOrDevice(device,3);
+      cy.addOrRmoveDataPoint(flowDataPointLabel, 'add');
+      cy.get(addDataPointsButton).click();
+      cy.get(`[title="${flowDataPointLabel}"]`).should('contains.text', flowDataPointLabel);
+      cy.get(computedPropertyConfigSaveButton).click();
+    };
+
+    // On configuring last measurement property,user should be able to see computed property configuration window
+    // User should be able to see "add data point "button and on click which would display will open up data point selector window.
+    // on data point selector window user should be able to see all the data points and all the devices.
+    // On selection of device,user should be able to see available data points.
+    // User can select only one data point for a selected device.
+    // User should get an error message "At maximum 1 active data points are allowed to be selected" when user selects multiple data point. Only one data point can be selected for a particular device.
+    // Verify the presence of toggle buttons for each of the device. User must be able to toggle off and on to remove/add the data point.
+    // Verify that the save button gets enabled after selecting only one data point.
+    it('DevicePropertyMeasurement_TC_015-1', () => {
```

Comment:
Rename the text case name. Refer other test cases.

Reply:
incorporated.